cicecore/cicedyn/general/ice_step_mod.F90

```diff
@@ -836,26 +847,26 @@ subroutine update_state (dt, daidt, dvidt, dvsdt, dagedt, offset)
                                    nt_strata     = nt_strata(:,:),   &
                                    Tf            = Tf(i,j,iblk))
 
-            if (present(offset)) then
-
-            !-----------------------------------------------------------------
-            ! Compute thermodynamic area and volume tendencies.
-            !-----------------------------------------------------------------
-
-               if (present(daidt)) daidt(i,j,iblk) = (aice(i,j,iblk) - daidt(i,j,iblk)) / dt
-               if (present(dvidt)) dvidt(i,j,iblk) = (vice(i,j,iblk) - dvidt(i,j,iblk)) / dt
-               if (present(dvsdt)) dvsdt(i,j,iblk) = (vsno(i,j,iblk) - dvsdt(i,j,iblk)) / dt
-               if (present(dagedt) .and. tr_iage) then
-                  if (offset > c0) then                 ! thermo
-                     if (trcr(i,j,nt_iage,iblk) > c0) &
-                     dagedt(i,j,iblk) = (trcr(i,j,nt_iage,iblk) &
-                                      - dagedt(i,j,iblk) - offset) / dt
-                  else                                  ! dynamics
-                     dagedt(i,j,iblk) = (trcr(i,j,nt_iage,iblk) &
-                                      - dagedt(i,j,iblk)) / dt
-                  endif
-               endif ! tr_iage
-            endif ! present(offset)
+         if (present(offset)) then
+
+      !-----------------------------------------------------------------
+      ! Compute thermodynamic area and volume tendencies.
+      !-----------------------------------------------------------------
+
+         if (present(daidt)) daidt(i,j,iblk) = (aice(i,j,iblk) - daidt(i,j,iblk)) / dt
```

Comment:
Why was the alignment changed in this block of code?  Is this from an older version of the code?